docs/contracts/sharing.rst

```diff
@@ -608,7 +610,7 @@ Parameters
 Returns
 -------
 
-``Promise`` returns ``void``: resolved when done
+``Promise`` returns ``any``: sharings as an object
```

Comment:
Can we please explain how this sharings object is structured?
E.g. 'Sharings as an object, where the top level entries are sha3(accounts), and each top level entry has sha3(sharedProperty) as an entry, and so on

The idea is that somebody using this gets an idea what the entries of the sharings object mean, because it can be confusing if you have never worked with it

Reply:
Related to comment above "link to to top section from getSharings to explain result of `getSharings`", would the explanation at the top of the documentation page help you or would you like to see some more details?
If you want more, where, at the top or in at the functions related functions?

Reply:
Ah my bad, I did not know this existed. That really helps, but as you stated, a link to this should really be added.